intel-retail/automated-self-checkout
# Feat: Add Visualizer containers to the pipeline server

## PR Checklist



## What are you changing?

Adding visualizer to the pipeline server

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- make run-pipeline-server
- go to localhost:3000 to see grafana visualizer

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
